templates/workspaces/base/terraform/variables.tf

```diff
@@ -102,5 +102,6 @@ variable "workspace_owner_object_id" {
   default     = ""
   description = "The Object Id of the user that you wish to be the Workspace Owner. E.g. the TEST_AUTOMATION_ACCOUNT."
 }
-
-
+variable "arm_use_msi" {
+  type = bool
+}
```

Comment:
Looks like this new var isn't used. Am I missing something?

Reply:
Dropping it made the porter az mixin fail, can it be related?